server/controllers/shared/referralsController.test.ts

```diff
@@ -153,12 +161,14 @@ describe('ReferralsController', () => {
 
         assertSharedDataServicesAreCalledWithExpectedArguments()
 
+        expect(SentenceInformationUtils.detailsSummaryListRows).toHaveBeenCalledWith(
+          sharedPageData.person.sentenceStartDate,
+          offenderSentenceAndOffences.sentenceTypeDescription,
+        )
         expect(response.render).toHaveBeenCalledWith('referrals/show/sentenceInformation', {
           ...sharedPageData,
-          detailsSummaryListRows: SentenceInformationUtils.detailsSummaryListRows(
-            sharedPageData.person.sentenceStartDate,
-            offenderSentenceAndOffences.sentenceTypeDescription,
-          ),
+          detailsSummaryListRows,
+          hasSentenceDetails: true,
```

Comment:
I may have missed something, unless this has slipped in from another commit?

Reply:
Ah, I think it should have come in `Display message when there are no sentence details`

Reply:
Oops